packages/contentstack-import/src/import/modules/global-fields.ts

```diff
@@ -140,4 +159,51 @@ export default class ImportGlobalFields extends BaseClass {
         });
     });
   }
+
+  async replaceGFs(): Promise<any> {
+    const onSuccess = ({ response: globalField, apiData: { uid } = { uid: null } }: any) => {
+      this.createdGFs.push(globalField);
+      this.gFsUidMapper[uid] = globalField;
+      fsUtil.writeFile(this.gFsUidMapperPath, this.gFsUidMapper);
+      log(this.config, 'Global field ' + uid + ' replaced successfully', 'success');
+    };
+
+    const onReject = ({ error, apiData: { uid } }: any) => {
+      log(this.importConfig, `Global fields '${uid}' failed to replace`, 'error');
+      log(this.importConfig, formatError(error), 'error');
+      this.failedGFs.push({ uid });
+    };
+
+    await this.makeConcurrentCall(
+      {
+        apiContent: this.existingGFs,
+        processName: 'Replace global fields',
+        apiParams: {
+          serializeData: this.serializeReplaceGFs.bind(this),
+          reject: onReject.bind(this),
+          resolve: onSuccess.bind(this),
```

Comment:
[MINOR] Need not bind the current object into it. However, the current code will still work the way we expect.

Reply:
@antonyagustine will take this on phase2, since this is done for all modules, we need to update everywhere and test again